Essentials/src/main/resources/messages.properties

```
@@ -863,8 +863,10 @@ noLocationFound=<dark_red>No valid location found.
 noMail=<primary>You do not have any mail.
 noMailOther=<secondary>{0} <primary>does not have any mail.
 noMatchingPlayers=<primary>No matching players found.
+noMetaComponents=Data Components are not supported in this version of Bukkit. Please use JSON NBT metadata.
 noMetaFirework=<dark_red>You do not have permission to apply firework meta.
 noMetaJson=JSON Metadata is not supported in this version of Bukkit.
+noMetaNbtKill=JSON NBT metadata is no longer supported, you must manually convert your defined items to data components. You can use Paper's tool to convert here: https://docs.papermc.io/misc/tools/item-command-converter
```

```suggestion
noMetaNbtKill=JSON NBT metadata is no longer supported. You must manually convert your defined items to data components. You can convert JSON NBT to data components here: https://docs.papermc.io/misc/tools/item-command-converter
```
Slightly more explicit